lib/tasks/one_time_changes.rake

```diff
@@ -0,0 +1,13 @@
+namespace :one_time_changes do
+  desc "A one time task responding to Nov 2019 request to update loan date fields."
+  task adjust_loan_dates: :environment do
+    Loan.all.each do |l|
```

Comment:
I like to use `find_each` in cases like this, IIRC there's some kind of performance benefit to doing so.

Reply:
thanks!